src/Geometry.cpp

```diff
@@ -1,10 +1,19 @@
 #include "Geometry.h"
 
-Geometry::Geometry(std::vector<glm::vec2>& v, std::vector<glm::vec3>& c)
+Geometry::Geometry(const Geometry& g)
+  : vertexBuffer(g.vertexBuffer)
+  , colourBuffer(g.colourBuffer)
+  , vertexArray(g.vertexArray)
+  , vertices(g.vertices)
+  , colours(g.colours)
+{
+}
+
+Geometry::Geometry(std::vector<glm::vec2> v, std::vector<glm::vec3> c)
   : vertexBuffer(0)
   , colourBuffer(0)
   , vertexArray(0)
-  , vertices(v)
-  , colours(c)
+  , vertices(std::move(v))
+  , colours(std::move(c))
 {
-}
\ No newline at end of file
+}
```

Comment:
Also, let's add newlines at the end of each new file.

To set this in VSCode:
* File->Preferences->Settings
* Search for "files.insertFinalNewline" and hover mouse
* Click the Edit button (Pencil icon on the left)
* Set to true
* Save the right window